### PR TITLE
Profile attributes management dev

### PR DESCRIPTION
**Pull request by vstanojevic**
Profile management v 1.0

### CODE REVIEW

**Comment by aleksandarsimonovic** [comment]
I have read the CLA Document and I hereby sign the CLA

File: knowage-vue/src/i18n/en_US.json [hunk 1]
```diff
@@ -127,6 +128,44 @@
...
+            "title": "Profile Attributes",
+            "name": "Name",
+            "description": "Description",
+            "validation": {
+              "extendedAlphanumeric": "Invalid format",
+              "maximumCharacters": "The maximum length allowed is {length}",
+              "required": "{field} is required"
+            },
+            "unsavedChangesHeader": "Unsaved changes",
+            "unsavedChangesMessage": "You have not saved your recent changes. Continue without saving ?",
```
Comment (Redjaw) [inline knowage-vue/src/i18n/en_US.json:141]: This label should be common in most list interfaces, move to a common.unsaved section

File: knowage-vue/src/i18n/en_US.json [hunk 2]
```diff
@@ -127,6 +128,44 @@
...
+            "valueDescription": "Value description",
+            "confirmDeleteMessage": "Are you sure you want to delete the selected {item}?",
+            "confirmUnsavedChangesMessage": "You have not saved your recent changes. Continue without saving?",
```
Comment (Redjaw) [inline knowage-vue/src/i18n/en_US.json:144]: This label should be common in most list interfaces, move to a common.confirm section

**Review by Redjaw — CHANGES_REQUESTED** [review]
Common message prompt translations can me put in a common section to avoid duplication between components. If one will be needed differently we can however use it.

File: knowage-vue/src/i18n/en_US.json (hunk 1 above)
Comment (vstanojevic) [reply knowage-vue/src/i18n/en_US.json:141]: Sorry, for to write you during your time off, but we already have the messages for confirm and unsaved changes in common.toast section and we have used it in roles-management components. Do you want as to remove it from that section and put it where you requested (common.unsaved and common.confirm)

Comment (Redjaw) [reply knowage-vue/src/i18n/en_US.json:141]: No sorry, you can use the one you already got here.
image
Just avoid proliferation when present.

Comment (vstanojevic) [reply knowage-vue/src/i18n/en_US.json]: This should be resolved

File: knowage-vue/src/i18n/en_US.json (hunk 2 above)
Comment (vstanojevic) [reply knowage-vue/src/i18n/en_US.json]: This should be resolved

**Review by Redjaw — APPROVED** [review]

**Comment by Redjaw** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**